accelerator/apis.h

```diff
@@ -28,14 +28,16 @@ extern "C" {
  * Generate and return an unused address from the seed. An unused address means
  * the address does not have any transaction with it yet.
  *
+ * @param[in] config IOTA API parameter configurations
```

Comment:
I am wondering if we can eliminate the use of `config` in parameter list? I thought the configuration should be hidden in certain handler.